Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelection.cpp

```diff
@@ -845,6 +940,24 @@ namespace AzToolsFramework
         EditorVertexSelectionBase<Vertex>::SetState(EditorVertexSelectionBase<Vertex>::State::Selecting);
     }
 
+    template<typename Vertex>
+    void EditorVertexSelectionVariable<Vertex>::DuplicateSelectedVertices()
+    {
+        DuplicateSelected();
+    }
+
+    template<typename Vertex>
+    void EditorVertexSelectionVariable<Vertex>::DeleteSelectedVertices()
+    {
+        DestroySelected();
+    }
+
+    template<typename Vertex>
+    void EditorVertexSelectionVariable<Vertex>::ClearVertexSelection()
+    {
+        EditorVertexSelectionBase<Vertex>::ClearSelected();
+    }
```

Comment:
question: Just curious why we need to add these aliases? 🤔 Is it purely for readability?